### ISSUE TITLE
Compiler: Encounter Unhandled Exception 0xC00000FD

### ISSUE THREAD

**Issue by KeitoTobi1** [vlang/v#14776]
**V version:**  V 0.2.4 01fdd5d
**OS:** Windows 11

I am Implemented [Monkey](https://monkeylang.org/) as Vlang.

Source Code is [this](https://github.com/vlang/v/files/8927233/monkey.zip).

When I was testing the execution, I was confronted with this error:

```
D:\Projects\V\monkey>v tests\parser_precedense_test.v
Unhandled Exception 0xC00000FD
```

**Comment by KeitoTobi1** [comment]
I made adjustments based on the V language code and it cured the problem.

**Comment by KeitoTobi1** [comment]
It has resolved itself, please see #15023 for details.
